packages/node/src/index.ts

```diff
@@ -376,9 +377,6 @@ function getAWSLambdaHandler(entrypoint: string, config: Config) {
   return '';
 }
 
-// Ensures that everything from `types.ts` is exported in the final `index.d.ts` file.
-export * from './types';
```

Comment:
I think this part is required so that users can

```ts
import type { VercelRequest } from '@vercel/node'`
```

Reply:
This is still supported. See in the `build.mjs` script that `src/types.d.ts` is copied to `dist/index.d.ts` which makes those types work for end-users.